source/core/ExecutionContext.cpp

```diff
@@ -298,6 +298,8 @@ ExecutionContext::ExecutionContext()
     _breakoutCount = 0;
     _runningQueueElt = static_cast<VIClump*>(nullptr);
     _timer._observerList = nullptr;
+     DebuggingContext debugContext;
+     debuggingContext = &debugContext;
```

Comment:
Why not just new the debuggingContext directly and then free it in the destructor? Anybody accessing the debuggingContext will have to be careful of the lifetime anyway.

Reply:
Done